package.json

```diff
@@ -25,6 +25,8 @@
     "types-publisher": "Microsoft/types-publisher#production"
   },
   "dependencies": {
-    "jslint": "^0.10.3"
+    "bodybuilder": "^2.2.0",
```

Comment:
Revert edits to the root `package.json` - if your package depends on type from another not-DT package, then your package's folder should have a `package.json` listing those type dependencies. The `test` file isn't for runtime tests - it's for typechecker tests - making sure objects typecheck correctly.